boards/Protectli/VP66xx/v0.9.0-results.csv

```diff
@@ -0,0 +1,125 @@
+Test name,Test ID,VP66x0 VP6650 v0.9.0,VP66x0 VP6670 v0.9.0
```

Comment:
Duplicated platform names found. Keep only the specific names, e.g., VP6670.

Reply:
https://github.com/Dasharo/osfv-results/compare/712c7f44d1f2f034f255e421fdb95b67f7b06dac..3f146cb93a885a176c2356bb9338e4e940eb7820#diff-1b63c0b8899f3331743d48ae393b810c41e045ec5f13c867f0a1e6904d694d0cL1-R1

Downloading the results automatically requires the column names to be exactly the same as on the test matrix. After downloading them they can be changed, but updating the file using the script would require changing them again.